src/commands/format.ts

```diff
@@ -0,0 +1,21 @@
+import { CommandDef } from './command-def';
+import { Message } from 'discord.js';
+import { addFormatting } from './add-formatting';
+
+export const format: CommandDef = {
+  prefix: 'format',
+  description: 'Formats the code contained in the message ID.',
+  command: async (message: Message): Promise<void> => {
+    const messageID = message.content.split(' ')[2];
```

Comment:
Verify the amount is as expected, as if the content passed isn't long enough this code will explode.

Reply:
Added a check for length